x-pack/test/apm_api_integration/tests/error_rate/service_apis.spec.ts

```diff
@@ -67,6 +67,16 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             ...commonQuery,
             kuery: `processor.event : "${processorEvent}"`,
```

Comment:
Do we still need this, now we are able to specify the `documentType`?

Reply:
`GET /internal/apm/services/{serviceName}/service_overview_instances/main_statistics` doesn't use `documentType` (yet).